MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,572 @@
+---
+layout: post
+title: Toolbar Customization in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about toolbar customization support in the Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Toolbar Customization in .NET MAUI Image Editor (SfImageEditor)
+
+You can customize the existing toolbar appearance, visibility, position, and orientation of each toolbar and can include new toolbars.
+
+## Toolbar style customization
+
+To customize the appearance of the toolbar by using the `ToolbarSettings` property. Below are some of the properties you can customize:
```

Comment:
You can customize the appearance of the toolbar by using the `ToolbarSettings` property. Below are some of the properties you can customize:

Reply:
Updated